jsEx/Form.js

```diff
@@ -0,0 +1,106 @@
+function validate() {
+  let username = document.getElementById("username").value;
+  let password = document.getElementById("password").value;
+  let confirmPassword = document.getElementById("confirm-password").value;
+
+  let isValidUser = isValidUsername(username);
+  let isValidPass = isValidPassword(password);
+  let isValidConfirmPass = isValidConfirmPassword(confirmPassword, password);
+
+  if (isValidUser) {
+    hideErrorMessag("invalidUsername");
+  }
+  if (isValidPass) {
+    hideErrorMessag("invalidPassword");
+  }
+  if (isValidConfirmPass) {
+    hideErrorMessag("invalidConfirmPassword");
+  }
+
+  return (isValidUser && isValidPass && isValidConfirmPass) ? request() : false;
+}
+
+function ajax(dataRequest) {
+  var xhr = new XMLHttpRequest();
+
+  xhr.open(dataRequest.method, dataRequest.url, true);
+
+  xhr.onreadystatechange = function() {
+    if (xhr.readyState === XMLHttpRequest.DONE && xhr.status === 200) {
+      dataRequest.success(xhr.response);
+    } else {
+      console.error(xhr.response);
+    }
+  };
+  
+  xhr.send(dataRequest.data);
+}
+
+var callback = function(jsonText) {
+  console.log(JSON.parse(jsonText)); 
+};
+
+function request() {
+  let username = document.getElementById("username").value;
+  let password = document.getElementById("password").value;
+  let confirmPassword = document.getElementById("confirm-password").value;
+
+  let data = {
+    "Username": username,
+    "Password": password,
+    "ConfirmPassword": confirmPassword
+  };
+
+  var dataJSON = JSON.stringify(data);
+  var dataRequest = {
+    data: dataJSON,
+    url: "http://localhost/WEB2020-php/jsEx/register.php",
```

Comment:
You may want to check here about how to make the url relative instead of using the domain hardcoded: https://github.com/tancheto/WEB2020-php/pull/6/files This way when someone check-outs your code on their machine and it is in a different folder it will still be working without any changes.